mDNSPosix/mDNSPosix.c

```diff
@@ -1721,14 +1764,13 @@ mDNSlocal void      PrintRoutingSocketMsg(const struct ifa_msghdr *pRSMsg)
 }
 #endif
 
-mDNSlocal mDNSu32       ProcessRoutingNotification(int sd)
+mDNSlocal void          ProcessRoutingNotification(int sd, GenLinkedList *changedInterfaces)
```

Comment:
There is `return mStatus_UnsupportedErr; // cannot decipher message` here but with this change, the function returns `void`. This makes a compile error.

Reply:
Indeed! It looks like that is only if USES_NETLINK is false. What platform are you on?

I think the solution is to just return like we do for the !USES_NETLINK version (see line 1690). I will push an update.

Reply:
I'm trying to port this to ESP32 and just noticed this.